src/Modal.js

```diff
@@ -267,6 +268,15 @@ const Modal = React.createClass({
       );
     }
 
+    const keys = Object.keys(_props);
+    const propTypes = Object.keys(Modal.propTypes);
+    const newProps = {};
+    keys.map(function (prop) {
+      if (!propTypes.includes(prop)) {
```

Comment:
`Array.prototype.includes` is not available in all browsers. That's why tests are failing. Separately this should be an arrow function though. Even more separately this should just be what @jquense said with `_.omit`.